scapy/fields.py

```diff
@@ -853,6 +853,11 @@ def i2repr(self, pkt, x):
         return lhex(self.i2h(pkt, x))
 
 
+class XLELongField(LELongField, XLongField):
```

Comment:
A similar field is already define in `scapy/layers/bluetooth.py`. Can you merge both definitions ?